SDPi_Supplement/asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc

```diff
@@ -0,0 +1,65 @@
+// = Use Case Feature: Synchronized Time Across Devices (STAD)
+
+#TODO:  Add use case metadata strategy#
+
+// [mdi_use_case#use_case_stad,actors='actor_somds_provider actor_somds_consumer',figure=vol2_figure_dev_24_probe_sequence,messages='message_announce_network_presence']
+
+// FROM TF-2 DEV-24 Transaction document
+// [sdpi_transaction#transaction_dev_24,actors='actor_somds_provider actor_somds_consumer',figure=vol2_figure_dev_24_probe_sequence,messages='message_announce_network_presence']
+
+[sdpi_offset=2]
+=== Use Case Feature {var_use_case_id}: Synchronized Time Across Devices (STAD)
+
+==== Narrative:
+Nurse Jean attaches a ventilator to the SDC based network in the ICU.  It automatically synchronizes its clock with the clocks of the other devices on the network.
+
+==== Technical View
+
+.Synchronized Time Across Devices (STAD) -- Technical View
+
+image::../../images/vol1-diagram-use-case-stad-tech-view.svg[align=center]
+
+==== Technical Pre-Conditions
+
+*Given* All devices communicate using SDC
+
+*And* NTP service on the SDC network
+
+==== Scenarios
+
+===== Scenario: STAD {var_use_case_id}.1 - Device is connected to network with NTP service
+
+*Given* Device has detected at least one NTP service
+
+*When* The NTP service is operational
+
+*Then* The device will synchronize its time with the NTP service time
+
+===== Scenario: STAD {var_use_case_id}.2 - Device is connected to network with a failed NTP server
+
+*Given* Device has not detected at least one NTP service
+
+*When* The NTP service is not operational or inaccessible
+
+*Then* The device will synchronize its time with another SDC device in the cluster
+
+===== Scenario: STAD {var_use_case_id}.3 - Device is connected to network with NTP server and a second device’s time setting is changed manually
+
+*Given* Device has detected at least one NTP service
+
+*When* The NTP service is operational
+
+*And* The time at another device is changed manually (maybe this is disallowed?)
+
+*Then* The device will continue to synchronize its time with the NTP service time
+
+===== Scenario: STAD {var_use_case_id}.4 - Device is connected to network with a failed NTP server and a second device’s time setting is changed manually
+
+*Given* Device has not detected at least one NTP service
+
+*When* The NTP service is not operational or inaccessible
+
+*And* The time at another device is changed manually
+
+*Then* The device will synchronize its time with the time from the SDC device that had its time changed
```

Comment:
As said, these mechanics do not exist at the moment.

Reply:
Would be ok for me. I just wanted to make clear that this use case (and others requiring to synchronize time with another SDC) are currently not implementable with SDC devices out-of-the-box

Reply:
Is that something that we should target for SDPi 1.1 or 1.x or ... 10.0?!

Reply:
Done when needed, I guess. :-)

Reply:
NOTE:  Decision was made to document in a Note Box at the start of any use cases / scenarios that are not implemented in SDPi 1.0.  Note also that as of last SDPi Friday's discussion (11.11.2022), there will be backlinks from profile use case sections to the Appendix C use case elements that the profile satisfies.  No forward links from the "generic" clinical use case itself.